cli/tests/containerize.bats

```diff
@@ -68,7 +68,7 @@ podman_xdg_vars_setup() {
   # These get unset by the preceding function call and must be restored in order
   # to use a single podman machine across the test run
   export XDG_CONFIG_HOME="$short_tmp_dir/.config"
-  export XDG_DATA_HOME="$short_tmp_dir/.local/share"
+  export XDG_DATA_HOME="$REAL_XDG_DATA_HOME"
```

Comment:
question blocking: does this break anything for Flox? What are we putting in XDG_DATA_HOME? Will we leave behind Flox stuff for every test run?

It looks like we may be able to use the `engine.env` section of containers.conf to override this for just podman https://docs.podman.io/en/stable/markdown/podman.1.html#environment-variables

Reply:
We set separate variables for Flox data e.g. FLOX_DATA_HOME, and those are put in temporary directories for the test run.

I'm not sure exactly what you're linking to, it just takes me to a section describing which environment variables control the storage locations, like XDG_DATA_HOME, and we're already using that. Note that the directory podman pulls container images to (`*/storage`) is not the same that it pulls the VM image to (`*/machine/applehv`).

We could set the environment variable just for the `podman machine init/start`) call though.

Reply:
> I'm not sure exactly what you're linking to, it just takes me to a section describing which environment variables control the storage locations, like XDG_DATA_HOME, and we're already using that. Note that the directory podman pulls container images to (`*/storage`) is not the same that it pulls the VM image to (`*/machine/applehv`).

`Podman can set up environment variables from env of [engine] table in containers.conf.`

I forgot about `FLOX_DATA_HOME` so I don't know if using container.conf is worth it or not